recipes/libmodbus/all/conanfile.py

```diff
@@ -1,15 +1,21 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from contextlib import contextmanager
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, get, replace_in_file, rename, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import unix_path, VCVars
 import os
 
-required_conan_version = ">=1.33.0"
+
+required_conan_version = ">=1.51.3"
 
 
 class LibmodbusConan(ConanFile):
     name = "libmodbus"
     description = "libmodbus is a free software library to send/receive data according to the Modbus protocol"
     homepage = "https://libmodbus.org/"
-    topics = ("conan", "libmodbus", "modbus", "protocol", "industry", "automation")
+    topics = ("libmodbus", "modbus", "protocol", "industry", "automation")
```

Comment:
```suggestion
    topics = ("modbus", "protocol", "industry", "automation")
```
both conan and the project name are not good keywords for searching.